[2023-10-02T12:54:55.496+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: launch_rocket.build_stage_2 manual__2023-10-02T12:54:53.545295+00:00 [queued]>
[2023-10-02T12:54:55.510+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: launch_rocket.build_stage_2 manual__2023-10-02T12:54:53.545295+00:00 [queued]>
[2023-10-02T12:54:55.511+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-10-02T12:54:55.541+0000] {taskinstance.py:1350} INFO - Executing <Task(BashOperator): build_stage_2> on 2023-10-02 12:54:53.545295+00:00
[2023-10-02T12:54:55.569+0000] {standard_task_runner.py:57} INFO - Started process 3508 to run task
[2023-10-02T12:54:55.599+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'launch_rocket', 'build_stage_2', 'manual__2023-10-02T12:54:53.545295+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag_exercise_1.py', '--cfg-path', '/tmp/tmpzyc4inwm']
[2023-10-02T12:54:55.606+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask build_stage_2
[2023-10-02T12:54:55.689+0000] {task_command.py:410} INFO - Running <TaskInstance: launch_rocket.build_stage_2 manual__2023-10-02T12:54:53.545295+00:00 [running]> on host f21dc12153d8
[2023-10-02T12:54:55.890+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='launch_rocket' AIRFLOW_CTX_TASK_ID='build_stage_2' AIRFLOW_CTX_EXECUTION_DATE='2023-10-02T12:54:53.545295+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-02T12:54:53.545295+00:00'
[2023-10-02T12:54:55.892+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-10-02T12:54:55.893+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo <Task(BashOperator): build_stage_2> is running in the <DAG: launch_rocket> pipeline']
[2023-10-02T12:54:55.959+0000] {subprocess.py:86} INFO - Output:
[2023-10-02T12:54:55.964+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `('
[2023-10-02T12:54:55.964+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo <Task(BashOperator): build_stage_2> is running in the <DAG: launch_rocket> pipeline'
[2023-10-02T12:54:55.975+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-10-02T12:54:56.009+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-10-02T12:54:56.020+0000] {taskinstance.py:1373} INFO - Marking task as FAILED. dag_id=launch_rocket, task_id=build_stage_2, execution_date=20231002T125453, start_date=20231002T125455, end_date=20231002T125456
[2023-10-02T12:54:56.050+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 10 for task build_stage_2 (Bash command failed. The command returned a non-zero exit code 2.; 3508)
[2023-10-02T12:54:56.083+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-10-02T12:54:56.150+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
